spec/helpers/produce_items_helper_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ProduceItemsHelper, type: :helper do
+  context 'when viewing the produce items index' do
+    let(:produce_item) { ProduceItem.create(name: 'carambola', category: 'fruit') }
+    let(:item_season_jan) { ItemSeason.create(produce_item_id: produce_item.id, month_index: 1, country_code: 'br') }
+    let(:item_season_fev) { ItemSeason.create(produce_item_id: produce_item.id, month_index: 2, country_code: 'br') }
+
+    describe '#formatted_month_name(produce, item season)' do
+      it 'adds / to the string, if its not the last or only month' do
+        expect(helper.formatted_month_list(produce_item, item_season_jan)).to eq('janeiro /')
+      end
+
+      it 'does not add / to the string, if its the last or only month' do
+        expect(helper.formatted_month_list(produce_item, item_season_fev)).to eq('fevereiro')
+      end
+    end
```

Comment:
**issue:** We should shift the view here. In many ways, I don't see that much of a point of doing a helper to attach `/` or not to the end of the string. It makes a bit more sense to just go and produce the end string (in this case `janeiro /fevereiro`. That way you could shift your tests to:

1. `when produce has no seasons`
2. `when produce has one season`
3. `when produce has multiple seasons`

Reply:
I have changed the helper, but I'm still not quite sure how I could write these tests